salt/utils/mac_utils.py

```diff
@@ -328,39 +349,59 @@ def _available_services(refresh=False):
                 # Follow symbolic links of files in _launchd_paths
                 file_path = os.path.join(root, file_name)
                 true_path = os.path.realpath(file_path)
-
+                log.trace('Gathering service info for {}'.format(true_path))
```

Comment:
Can you add a commit here to fix this logging? We shouldn't be using `.format()` in logging. And then on merge-forward, it will correct it in the other branches.